doc/source/getting_started/write_documentation.rst

```diff
@@ -89,6 +89,11 @@ Here's a short summary of how to write good documentation:
    consistent structure. Use headings, subheadings, and a table of contents if
    necessary to help users navigate your documentation easily.
 
+#. **Use sphinx tool properly**: Sphinx has multiple features and directives. It is
+   recommended you get yourself familiarize with them before starting to write.
+   Useful guidance can be found
+   `here <sphinx_directives_>`_, `here <sphinx_basics_>`_ and `here <docutils_directives_>`_.
```

Comment:
```suggestion
#. **Use Sphinx properly**: Sphinx has multiple features and directives. Before
   starting to write documentation, you should get familiar with it. For useful
   guidance, see `here <sphinx_directives_>`_, `here <sphinx_basics_>`_ and `here <docutils_directives_>`_.
```
Again--why I like a central links file, I also dislike a central links file. The Google dev style guide says to never use "here" as a link but to use informative text as your display link.